BenchmarkTest00222.py

```
@@ -0,0 +1,17 @@
+#{fact rule=code-injection@v1.0 defects=1}
+
+from flask import Flask
+
+app = Flask(__name__)
+
+def hello():
+  app.run()
+
+
+# ruleid:avoid_using_app_run_directly
+app.run(debug=True)
```

Ruff (0.13.1)

Use of `debug=True` in Flask app detected

(S201)